end_to_end/nlp_mlops_company_sentiment/03_nlp_company_earnings_analysis_lambda_async.ipynb

```diff
@@ -0,0 +1,1343 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # !pip install transformers
</pre><p>Consider uncommenting pip installs of transformers, typing, sentencepiece, fiscalyear... most customers will not already have them installed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>